gsa/src/web/pages/results/__tests__/detailspage.js

```diff
@@ -0,0 +1,464 @@
+      'href',
+      'test/en/reports.html#displaying-all-existing-results',
+    );
+
+    expect(screen.getAllByTitle('Results List')[0]).toBeInTheDocument();
+    expect(links[1]).toHaveAttribute('href', '/results');
+
+    expect(screen.getAllByTitle('Export Result as XML')[0]).toBeInTheDocument();
+    expect(screen.getAllByTitle('Add new Note')[0]).toBeInTheDocument();
+    expect(screen.getAllByTitle('Add new Override')[0]).toBeInTheDocument();
+    expect(screen.getAllByTitle('Create new Ticket')[0]).toBeInTheDocument();
+    expect(
+      screen.getAllByTitle('Corresponding Task (task 1)')[0],
+    ).toBeInTheDocument();
+    expect(screen.getAllByTitle('Corresponding Report')[0]).toBeInTheDocument();
+    expect(
+      screen.getAllByTitle('Corresponding Tickets')[0],
+    ).toBeInTheDocument();
+
+    // Header
+    expect(element).toHaveTextContent('Result: foo');
+    expect(element).toHaveTextContent('ID:12345');
+    expect(element).toHaveTextContent('Created:Sun, Jun 2, 2019 2:00 PM CEST');
+    expect(element).toHaveTextContent('Modified:Mon, Jun 3, 2019 1:00 PM CEST');
+    expect(element).toHaveTextContent('Owner:admin');
+
+    // Tabs
+    const tabs = screen.getAllByTestId('entities-tab-title');
+    expect(tabs[0]).toHaveTextContent('User Tags');
+
+    // Details
+    const heading = baseElement.querySelectorAll('h2');
+
+    expect(heading[1]).toHaveTextContent('Vulnerability');
+    expect(baseElement).toHaveTextContent('Namefoo');
+    expect(baseElement).toHaveTextContent('Severity5.0 (Medium)');
+    expect(baseElement).toHaveTextContent('QoD80 %');
+    expect(baseElement).toHaveTextContent('Host109.876.54.321');
+    expect(baseElement).toHaveTextContent('Location80/tcp');
+
+    expect(heading[2]).toHaveTextContent('Summary');
+    expect(baseElement).toHaveTextContent('This is a mock result');
+
+    expect(heading[3]).toHaveTextContent('Detection Result');
+    expect(baseElement).toHaveTextContent('This is a description');
+
+    expect(heading[4]).toHaveTextContent('Insight');
+    expect(baseElement).toHaveTextContent('This is just a test');
+
+    expect(heading[5]).toHaveTextContent('Detection Method');
+    expect(baseElement).toHaveTextContent('This is the detection method');
+    expect(baseElement).toHaveTextContent(
+      'Details: nvt1 OID: 1.3.6.1.4.1.25623.1.12345',
+    );
+    expect(baseElement).toHaveTextContent('Version used: 2019-02-14T07:33:50Z');
+
+    expect(heading[6]).toHaveTextContent('Affected Software');
+    expect(baseElement).toHaveTextContent('Affects test cases only');
+
+    expect(heading[7]).toHaveTextContent('Impact');
+    expect(baseElement).toHaveTextContent('No real impact');
+
+    expect(heading[8]).toHaveTextContent('Solution');
+    expect(baseElement).toHaveTextContent(
+      'Solution Type: st_mitigate.svgMitigation',
+    );
+    expect(baseElement).toHaveTextContent('Keep writing tests');
+
+    expect(heading[9]).toHaveTextContent('References');
```

Comment:
I would also add a test BID reference? Maybe also CERT and (if there are Other links) ... to cover all reference types